### PR TITLE
Top level statements tutorial

### LINKED ISSUE #20593: Tutorials: Top level statements
- The Tour of C# "hello world" example should show both formats, starting with the top level statements version.
- Provide an "explore new features" tutorial showing how to use top level statements for Azure functions.

### PR DESCRIPTION
**Pull request by BillWagner**
Fixes #20593 

Create tutorial for using top-level statements.

This tutorial enables readers to explore top level statements to learn the syntax rules for using top-level statements, learn to create script-like programs, and refactor those script-like programs into maintainable utilities.

### CODE REVIEW

File: docs/csharp/tutorials/exploration/top-level-statements.md [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+using System;
+
+Console.WriteLine(args);
+```
+
+You don't declare an `args` variable. For the single source file that contains your top-level statements, the compiler recognizes `args` to mean the command-line arguments. The type of args is a `string[]`, as in all C# programs.
+
+You can test your code by running the following `dotnet run` command:
+
+```console
```
Comment (Youssef1313) [inline docs/csharp/tutorials/exploration/top-level-statements.md:71]:
~~~suggestion
```dotnetcli
~~~

File: docs/csharp/tutorials/exploration/top-level-statements.md [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+
+:::code language="csharp" source="snippets/top-level-statements/Program.cs" ID="Answers":::
+
+This array has 12 answers that are affirmative, six that are non-committal, and six that are negative. Next, add the following code to generate and display a random answer from the array:
+
+:::code language="csharp" source="snippets/top-level-statements/Program.cs" ID="GenerateAnswer":::
+
+You can run the application again to see the results. You should see something like the following output:
+
+```console
```
Comment (CamSoper) [inline docs/csharp/tutorials/exploration/top-level-statements.md:99]:
```suggestion
```dotnetcli
```

**Review by CamSoper — DISMISSED** [review]

**Comment by BillWagner** [comment]
@CamSoper Thanks for the review. I've made all the requested updates. Can you approve?

File: docs/csharp/tutorials/exploration/top-level-statements.md [hunk 3]
```diff
@@ -0,0 +1,217 @@
+---
+title: Top-level statements - C# tutorial
+description: This tutorial shows how you can use top-level statements to experiment and prove concepts while exploring your ideas
+ms.date: 10/28/2020
+---
+# Tutorial: Explore ideas using top-level statements to build code as you learn
+
+Introduction and problem statement
```
Comment (pkulikov) [inline docs/csharp/tutorials/exploration/top-level-statements.md:8]: That line looks to be the remaining one from original outline.

File: docs/csharp/tutorials/exploration/top-level-statements.md [hunk 4]
```diff
@@ -0,0 +1,217 @@
...
+    }
+    Console.WriteLine();
+}
+```
+
+The preceding code creates a local function inside your main method. That's still not reusable. So, extract that code into a class. Create a new file named *utilities.cs* and add the following code:
+
+:::code language="csharp" source="snippets/top-level-statements/UtilitiesPassOne.cs" ID="SnippetUtilities":::
+
+. Top-level statements can only be in one file, and that file cannot contain namespaces or types.
```
Comment (pkulikov) [inline docs/csharp/tutorials/exploration/top-level-statements.md:205]: Typo:
```suggestion
Top-level statements can only be in one file, and that file cannot contain namespaces or types.
```

**Review by CamSoper — APPROVED** [review]

**Comment by gewarren** [comment]
@BillWagner This introduced a build warning.

**Pull request merged**